providers/openfeature-provider-flagd/src/openfeature/contrib/provider/flagd/resolvers/grpc.py

```diff
@@ -13,10 +13,13 @@
     TypeMismatchError,
 )
 from openfeature.flag_evaluation import FlagResolutionDetails
+from openfeature.schemas.protobuf.flagd.evaluation.v1 import (  # type:ignore[import-not-found]
```

Comment:
```suggestion
from openfeature.schemas.protobuf.flagd.evaluation.v1 import (
```

Removing this gives me no problems locally. I can run `hatch build` and `hatch test` without issue.

Reply:
because you have run `hatch build` if somebody edits documentation, or only commits something, ruff will complain, that those are not found

Reply:
try `hatch clean` modify a file and try to commit it, normally typechecking should kick in, and prevent you from commiting